GOVERNANCE.md

```diff
@@ -0,0 +1,158 @@
+
+## Decisionmaking
+
+### Team members 
+
+Team member status is given to those who have made ongoing contributions for at least 3 months to the Prometheus Project. This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support could also be taken into account. 
+
+New members may be proposed by any existing member by email to [prometheus-team][team] and voted on by [consensus](#consensus). If no consensus is reached, the new member is not accepted.
+
+Team members are added to the [GitHub organization][gh] and have commit rights to all projects. They should however respect the maintainers of each project.
+
+Team members may retire at any time by emailing [the team][team].
+
+Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team]. For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+
+Upon death of a member, their team membership ends automatically.
+
+Unless voted upon otherwise, membership changes take effect immediately. Unless voted upon otherwise, any membership change is announced to [the public developer mailing list][devs] as soon as it is in effect.
+
+### Maintainers
+
+Maintainers are team members who are lead one or more project(s) and serve as a point of conflict resolution amongst the contributors to this project.
+
+A maintainer or committer may resign by notifying the [team mailing list][team]. A maintainer with no project activity for a year is considered to have resigned. Maintainers who wish to resign are encouraged to propose another team member to take over the project.
+
+An extraordinary change of maintainership of any project may be proposed on the [team mailing list][team] and is voted on by [consensus](#consensus). If no consensus can be reached, the matter is resolved by [majority vote](#majority-vote).
+
+A project may have more multiple maintainers, as long as the responsibilities are clearly agreed upon between them. This includes coordinating who handles which issues and pull requests.
+
+Any maintainership change is performed by changing the project's MAINTAINERS.md on the main branch of the repository, and should be announced on the [developers mailing list][devs].
+
+### Technical decisions
+
+Technical decisions that only affect a single project are made informally by the maintainer of this project, and [lazy consensus](#consensus) is assumed.
+
+Technical decisions that span multiple parts of the Prometheus Project are discussed and made on the [Prometheus developer mailing list][devs]. Decisions are usually made by [consensus](#consensus). If no consensus can be reached, the matter may be resolved by [majority vote](#majority-vote).
+
+### New projects
+
+New projects under the Prometheus Project must be proposed to the [developer mailing list][devs] and are subject to [lazy consensus](#consensus).
```

Comment:
I pulled this out of the FAQ & from our past practice. The difference to pure lazy consensus is that they do need to be at least proposed via mail.

Reply:
This is also mentioned above, I'd remove the dupe.

I'd also suggest only using one of repo/project for clarity.

Reply:
ack